tools/scripts/BuildZippedCLI.ps1

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+<#
+.SYNOPSIS
+Builds a ZIP file that contains the Axe.Windows CLI. Output is named AxeWindowsCLI.zip
+
+.PARAMETER Configuration
+Debug or Release. Will retain PDB files in the ZIP file if DEBUG. Use $(ConfigurationName) if calling from a csproj file
+
+.PARAMETER TargetDir
+The directory where the files were built (and where the ZIP file will be dropped). Use $(TargetDir) if calling from a csproj file
+
+.Example Usage
+.\BuildZippedCLI.ps1 -Configuration release -TargetDir c:\myrepo\src\CLI\bin\release\netcoreapp3.0
+#>
+
+param(
+    $Configuration,
+    $TargetDir
+)
+
+# Uncomment the next line for debugging
```

Comment:
I recommend using the following boilerplate at the top of any powershell script; it will catch things like typos in variable names and fail fast rather than silently if non-terminating errors you aren't expecting get emitted:

```powershell
Set-StrictMode -Version Latest
$script:ErrorActionPreference = 'Stop'
```

Reply:
Thanks--added